Sortable.js

```diff
@@ -1520,11 +1496,9 @@
 							newIndex = oldIndex;
 						}
 
-						if (options.swap)
+						if (options.swap && lastSwapEl)
```

Comment:
We do want `onEnd` to call, even if they do not have a swap target

Reply:
Makes sense to me!